plugins/dynatrace/src/api/DynatraceClient.ts

```diff
@@ -52,11 +57,31 @@ export class DynatraceClient implements DynatraceApi {
     );
   }
 
+  async getDynatraceSyntheticFailures(
+    syntheticsId: string,
+  ): Promise<DynatraceSyntheticResults | undefined> {
+    if (!syntheticsId) {
+      throw new Error('Dynatrace syntheticId is required');
+    }
+
+    return this.callApi(`synthetic/execution/${syntheticsId}/FAILED`, {});
```

Comment:
Always remember to encodeURIComponent things that are forming URL paths, to handle malicious / malformed inputs. Same below